app.js

```diff
@@ -1,13 +1,15 @@
 const express = require("express");
 const app = express();
+app.use(express.json())
 const endpoints = require("./endpoints.json");
-// const { api , getApiTopics } = require("./controller")
+const { api, getApiTopics } = require("./controller");
 
-app.get("/api", (req, res) => {
-  console.log("inside app")
+app.get("api", (req, res) => {
```

Comment:
we don't need this endpoint - we want to match /api which is what you've got below - this endpoint will never be reached as it doesn't start with /